DQM/L1TMonitorClient/python/L1TStage2uGTClient_cff.py

```diff
@@ -26,12 +26,16 @@
        "Ratio_First_Bunch_In_Train 'First Bunch In Train Ratio; Bunch crossing number relative to L1A; Algorithm trigger bit' first_bunch_in_train den_first_bunch_in_train",
        "Ratio_Last_Bunch_In_Train 'Last Bunch In Train Ratio; Bunch crossing number relative to L1A; Algorithm trigger bit' last_bunch_in_train den_last_bunch_in_train",
        "Ratio_Isolated_Bunch_In_Train 'Isolated Bunch In Train Ratio; Bunch crossing number relative to L1A; Algorithm trigger bit' isolated_bunch den_isolated_bunch_in_train",
+       "Ratio_Prescaled_First_Bunch_In_Train 'First Bunch In Train Ratio for Prescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' prescaled_first_collision_run_ den_prescaled_first_collision_run_",
+       "Ratio_Prescaled_Isolated_Bunch 'Isolated Bunch Ratio for Prescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' prescaled_isolated_collision_run_ den_prescaled_isolated_collision_run_",
+       "Ratio_Prescaled_Last_Bunch_In_Train 'Last Bunch In Train Ratio for Prescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' prescaled_last_collision_run_ den_prescaled_last_collision_run_",
+       "Ratio_Unprescaled_First_Bunch_In_Train 'First Bunch In Train Ratio for Unprescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' unprescaled_first_collision_run_ den_unprescaled_first_collision_run_",
+       "Ratio_Unprescaled_Isolated_Bunch 'Isolated Bunch Ratio for Unprescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' unprescaled_isolated_collision_run_ den_unprescaled_isolated_collision_run_",
+       "Ratio_Unprescaled_Last_Bunch_In_Train 'Last Bunch In Train Ratio for Unprescaled Triggers; Bunch crossing number relative to L1A; Algorithm trigger bit' unprescaled_last_collision_run_ den_unprescaled_last_collision_run_",
     ),
     resolution = cms.vstring(),
     outputFileName = cms.untracked.string(""),
     verbose = cms.untracked.uint32(0),
```

Comment:
Hi @dtsitson 
there seems to something wrong with this diff that github shows. Because what is in the 10_2_X branch is not what is shown on the left side here https://github.com/cms-sw/cmssw/blob/CMSSW_10_1_X/DQM/L1TMonitorClient/python/L1TStage2uGTClient_cff.py#L24-L36 . Can you please double check that what you have in your local branch has the configuration as in cms-sw/CMSSW_10_1_X plus the six additional lines with the new ratio plots?